src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -266,7 +270,8 @@ def corrected_voronoi_diagram(points):
         V = VoronoiDiagram(configuration)
         valid = True
         for r in V.regions().items():
-            if not r[1].rays() and not r[1].interior_contains(apprpoints[r[0].affine()]):
+            if not r[1].rays() and \
+               not r[1].interior_contains(apprpoints[r[0].affine()]):
```

Comment:
```suggestion
            if (not r[1].rays() and
                not r[1].interior_contains(apprpoints[r[0].affine()])):
```
Python style recommends this to help avoid potential errors.